mantidproject/mantid
# SingleCrystalDiffuseReduction workflow algorithm

This is a workflow algorithm for the most common reduction done on Corelli, but it should also work elsewhere, I did test it will Topaz data.

**To test:**
 * Look at the usage examples, anyone at SNS should be able to run them

A systemtest was added as it is a workflow algorithm and not fast enough for a unit test. Additional systemtests will be added later with more complete data and run for SNS only.

Release notes have been added

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
